File: 1 - Iniciante/C99/BEE1074.c
#include <stdio.h>

int main(void){
	int i, qtd, num;
	
	scanf("%d", &qtd);
	
	for (i=1; i <= qtd; i++){
		
		scanf("%d", &num);
		
		if (num>0 && num % 2 == 0){
			printf("EVEN POSITIVE\n");
		} else if (num>0 && num % 2 != 0){
			printf("ODD POSITIVE\n");
		} else if (num<0 && num % 2 == 0){
			printf("EVEN NEGATIVE\n");
		} else if(num<0 && num % 2 != 0){
			printf("ODD NEGATIVE\n");
		} else printf("NULL\n");
		
	} 
	
	return 0;
}

File: 1 - Iniciante/Python 3.11/BEE1074.py
def validation(num):
    if num == 0:
        return 'NULL'
    elif (num%2==0 and num>0):
        return 'EVEN POSITIVE'
    elif (num%2==0 and num<0):
        return 'EVEN NEGATIVE'
    elif (num%2!=0 and num>0):
        return 'ODD POSITIVE'
    else:
        return 'ODD NEGATIVE'

qt = int(input())
for i in range(qt):
    print(validation(int(input())))